CHANGELOG.md

```diff
@@ -1,3 +1,124 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta3
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` which distributes tables
+  without blocking writes
+
+* Adds `citus_split_shard_by_split_points()` function
+
+* Adds support for non-blocking tenant isolation
+
+* Separates columnar storage AM into a separate logical extension
+
+* Supports logical replication in `replicate_reference_tables()`
+
+* Improves performance of blocking shard moves
+
+* Uses a faster custom copy logic for non-blocking shard moves
```

Comment:
```suggestion
* Improves non-blocking shard moves with a faster custom copy logic
```

We do not have much `Uses ..` entries. This is an improvement to an already existing feature I guess.

Reply:
agreed